### ISSUE TITLE
Is it possible to extend to the case of dual arms?

### ISSUE THREAD

**Issue by JiaoxianDu** [SYSU-STAR/REMANI-Planner#4]
May I ask if the framework supports extension to dual-arm planning, similar to humanoid upper-body robots?

**Comment by CK1201** [comment]
Yes, it is definitely possible to extend the framework to support dual-arm planning.
